nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -14,9 +14,11 @@ nextflow.preview.dsl = 2
 /*
  * Print help message if required
  */
-include { print_help; check_genome; check_hostname; check_awsbatch; create_summary; send_email } from './modules/local/functions'
 if (params.help) {
-    print_help()
+    // TODO nf-core: Update typical command used to run pipeline
+    def command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv -profile docker"
```

Comment:
Might be clearer to hardcode the pipeline name here as the developer is going to have to edit manually anyway..

```suggestion
    def command = "nextflow run {{ cookiecutter.name }} --input samplesheet.csv -profile docker"
```

Reply:
Fixed https://github.com/nf-core/tools/pull/665/commits/feddeb68aa3655f3c9f736b2df12974b0f2368a8